service/commons/storable/internal/src/main/java/org/eclipse/kapua/service/storable/model/AbstractStorableListResult.java

```diff
@@ -53,6 +53,7 @@ public AbstractStorableListResult(List<E> storables, Long totalCount) {
 
         addItems(storables);
         setTotalCount(totalCount);
+        this.limitExceeded = totalCount > storables.size();
```

Comment:
This is not actually correct in the case of the last page. 
With:

- 120 total count
- 100 offset
- 50 limit

this will make `limitExceeded` true with only 20 results, so the `limitExceeded` should be false.